src/features/prompts/components/new-prompt-button.tsx

```diff
@@ -229,6 +250,28 @@ export const NewPromptForm = (props: {
             </>
           )}
         />
+        <FormField
+          control={form.control}
+          name="config"
+          render={({ field }) => (
+            <FormItem>
+              <FormLabel>Config</FormLabel>
+              <JsonView
```

Comment:
I just checked this and I was not able to create a nested json. Could you look into this?